go.mod

```diff
@@ -106,6 +106,7 @@ replace (
 	github.com/kubevirt/terraform-provider-kubevirt => github.com/nirarg/terraform-provider-kubevirt v0.0.0-20201222125919-101cee051ed3
 	github.com/metal3-io/baremetal-operator => github.com/openshift/baremetal-operator v0.0.0-20200715132148-0f91f62a41fe
 	github.com/metal3-io/cluster-api-provider-baremetal => github.com/openshift/cluster-api-provider-baremetal v0.0.0-20190821174549-a2a477909c1d
+	github.com/open-cluster-management/multicloud-operators-foundation => github.com/stolostron/multicloud-operators-foundation v0.0.0-20210824054957-51557a572f11
```

Comment:
For the main branch, we recommend updating to the latest revision.

Reply:
updated